rviz_plugins/eStop_plugin/src/bare_panel.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BARE_PANEL_H
+#define BARE_PANEL_H
+
+#include <ros/ros.h>
+#include <QLabel>
+#include <std_msgs/Bool.h>
+
+#include <rviz/panel.h>
+
+class QLineEdit;
+
+namespace bare
+{
+
+  class BarePanel: public rviz::Panel
+  {
+    Q_OBJECT
+    public:
+      BarePanel( QWidget* parent = 0 );
+
+    private:
+      void subCallback(const std_msgs::Bool &msg);
+
+    public Q_SLOTS:
+
+    protected Q_SLOTS:
+
+    protected:
+      ros::Subscriber sub;
+      ros::NodeHandle nh_;
```

Comment:
There are two conventions for private or protected members in the code right now are either a single underscore preceding the member name or no extra punctuation added to the member name. (ie. _nh or nh).
Please change these names to match one of these conventions and make sure you only use one for the whole file.